src/storage/src/table/cell_based_table.rs

```diff
@@ -129,21 +129,22 @@ impl<S: StateStore, SER: RowSerializer> CellBasedTableBase<S, SER, READ_ONLY> {
     /// set of `column_ids`. The output will only contains columns with the given ids in the same
     /// order.
     /// This is parameterized on cell based row serializer.
+    // TODO: allow specifying the distribution keys and vnodes.
```

Comment:
`CellBasedTable` is directly constructed only in batch query executor or seq scan. We leave them untouched, while specifying vnodes and dist keys for streaming-purpose `StateTable` is implemented.